docker/openemr/5.0.2/openemr.conf

```diff
@@ -1,40 +1,33 @@
 LoadModule rewrite_module modules/mod_rewrite.so
 LoadModule allowmethods_module modules/mod_allowmethods.so
 
-
-# The ServerName directive sets the request scheme, hostname and port that
-# the server uses to identify itself. This is used when creating
-# redirection URLs. In the context of virtual hosts, the ServerName
-# specifies what hostname must appear in the request's Host: header to
-# match this virtual host. For the default virtual host (this file) this
-# value is not decisive as it is used as a last resort host regardless.
-# However, you must set it for any further virtual host explicitly.
-#ServerName www.example.com
-
-
 ## Security Options
+# Strong HTTP Protocol
+HTTPProtocolOptions Strict
 Protocols http/1.1
-# Removes revealing HTTP Headers
+        
+# Don't Reveal Server
 ServerSignature off
 ServerTokens Prod
-TraceEnable Off
+Header unset Server
+
+# No ETag
 FileETag None
 Header unset ETag
+
 # Set HSTS and X-XSS protection
 Header set Strict-Transport-Security "max-age=31536000; includeSubDomains; preload"
 Header set X-XSS-Protection "1; mode=block"
-ServerAdmin webmaster@localhost
+
 # Narrow document root
 DocumentRoot /var/www/localhost/htdocs/openemr
 ErrorLog ${APACHE_LOG_DIR}/error.log
 CustomLog ${APACHE_LOG_DIR}/access.log combined
 
 
     <Directory /var/www/localhost/htdocs/openemr>
-        # Only allow these HTTP Methods
-        AllowMethods GET POST HEAD OPTIONS
-        # No indexes anywhere
-        Options -Indexes
+        AllowMethods GET POST #Only allow these HTTP Methods
```

Comment:
Should not do this line (note I have had to add PUT and DELETE to this so far for the api support). Guessing that HEAD and OPTIONS can be removed though. thoughts @danehrlich1 ?

Reply:
btw, regarding HEAD, appears good idea to keep for caching of static files (like assets, for example)